modules/core/src/test/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationStatisticsTest.java

```diff
@@ -53,7 +60,7 @@
     private final Object mux = new Object();
 
     /** */
-    private final CountDownLatch latch = new CountDownLatch(1);
+    private CountDownLatch latch = new CountDownLatch(1);
```

Comment:
Why this change?

Reply:
Artifact from test that didn't make it to the final version. I'll revert it.